Readme.md

```diff
@@ -1,10 +1,18 @@
+<div id="header" align="center">
+
```

Comment:
@AgisilaosVasileiadis Add the build status icon in before the Issues button :) The same on as on Cobol repo (Tomas did it there, so ask him if you need help)